mne/decoding/transformer.py

```diff
@@ -170,7 +170,7 @@ def transform(self, epochs_data, y=None):
         memory usage may be large with big data.
         """
         return _sklearn_reshape_apply(self._scaler.transform, True,
-                                      epochs_data, y)
+                                      epochs_data)
```

Comment:
Here is the error:

https://travis-ci.org/mne-tools/mne-python/jobs/263653811#L2857

Maybe we need to pass as a `**kwargs`, i.e. `y=y` instead of as positional `*args`?